src/status_im/android/core.cljs

```diff
@@ -169,7 +170,18 @@
                                        :contact-list-modal contact-list-modal)]
                        [component])]])]]))))})))
 
+(def exc (js/require "react-native-exception-handler"))
```

Comment:
Sure I will do that! Actually I will probably remove `react-native-exception-handler` as it does really little and go bare metal with `react-native` APIs.
Would it make sense to try to extract common ios/android behaviour in a shared namespace?